_posts/en/newsletters/2021-03-17-newsletter.md

```diff
@@ -0,0 +1,131 @@
+
+- [BTCPay Server 1.0.7.0][] FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21007][] bitcoind: Add -daemonwait option to wait for initialization FIXME: jnewbery
+
+- [C-Lightning #4404][] allows the `keysend` RPC (see [Newsletter
+  #107][news107 keysend]) to send messages even to nodes that don't
+  explicitly signal that they support the feature.  As
+  [discussed][fromknecht keysend], the signal was never standardized and
+  the procedure implemented by LND didn't depend on signaling, so this
+  change should allow C-Lightning to send to roughly the same set of
+  nodes tha LND can address.
+
+- [C-Lightning #4410][] brings the experimental implementation for
+  dual-funded channels into line with the most recent [draft
+  specification][bolts #851] changes.  Most notably, the use of Proof of
+  Discrete Log Equivalency (PODLE) has been dropped, at least
+  temporarily (see [Newsletter #83][news83 podle] for original
+  discussion of PODLEs and [Newsletter #131][news131 podle] for
+  discussion about alternatives).  Subsequent to this merge, a [new
+  PR][c-lightning #4427] was opened that will make experimentation with
+  dual-funding more accessible by eliminating the need to compile
+  C-Lightning with special build flags (although a special configuration
+  option will still be required).
+
+- [LND #5083][] allows a [PSBT][topic psbt] to be read from a file
+  rather than by reading the standard input (stdin) file descriptor.
+  Some terminals have a [limit][lnd #5080] on the number of characters
+  that can be added to stdin simultaneously (i.e. pasted), which made
+  PSBTs over 4096 characters (3 KiB) <!-- PSBTs are base64 encoded,
+  base64 provides 6 bits per character, so 4096 * 6 / 8 = 3072 = 3 KiB
+  --> unusable.  Especially now that several hardware wallets require
+  PSBTs include previous transactions for segwit spends (see [Newsletter
+  #101][news101 segwit overpayment]), it's easy to create PSBTs over 3
+  KiB in size.
+
+- [LND #5033][] adds an `updatechanstatus` RPC that can advertise that a
+  channel has been disabled (similar to your node going offline) or that
+  it's been re-enabled (similar to your node coming back online).
+
+- [Rust-Lightning #826][] increases the maximum allowed `OP_CHECKSEQUENCEVERIFY`
+  delay for to-self outputs in unilateral channel close transactions to 2016
+  blocks. This fixes an interoperability issue when opening large channels from
+  LND, which may request a `to_self_delay` up to 2016 blocks, larger than the
```

Comment:
"large channels _with_ LND" (or does this only affect channels that are opened by LND?)

Reply:
If I'm reading the code correctly, I believe this indeed only affects channels that are opened by LND.  When RL opens a channel, it only requests a delay of 144 blocks, which isn't changed in this PR.

That said, I think "*with* LND" covers both scenarios, so I'm going to change to that.  Happy to undo if @dongcarl or anyone has strong feelings about it.

Reply:
Looks good. Thanks!